### ISSUE TITLE
Event/Misc: Hallow's End

### ISSUE THREAD

**Issue by STARRHELD** [azerothcore/azerothcore-wotlk#691]
**Description**: Hallow's End

**Current behaviour**: This is an assemblage of the current issues with Hallow's End. Please report anything you notice.

**Expected behaviour**: --

**Steps to reproduce the problem**:

**Branch(es)**: Master

**AC hash/commit**:  612e996209ef+

**Operating system**:  Windows 7

**Modules**:  None

**Other Customizations**: None

[//]: # (This template is for problem reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)
[//]: # (For fixes containing c++ create a Pull Request)

**Comment by STARRHELD** [comment]
### List of issues with **Hallow's End**.

- [ ] The quest (**Call the Headless Horseman**) is missing the **RequestItemsText** and the **OfferRewardText**.
> Based on https://youtu.be/fC9ZCLzfl8Q?t=3m42s

**Comment by STARRHELD** [comment]
Fixes:
```SQL
-- Call of the Headless Horseman (11405)
UPDATE `quest_template` SET `RequestItemsText`='This dirt has been recently disturbed. Earthworms lie in the topsoil, squirming to escape the dark depths beneath them.' WHERE `ID`=11405;
UPDATE `quest_template` SET `OfferRewardText`='Will you lay the candle down and call the Headless Horseman from his doomed rest?' WHERE `ID`=11405;
```

**Comment by StygianTheBest** [comment]
I just tested again. If anyone knows a quick way to reset daily quests let me know. I did unbind and remove add quests but it didn't work for calling the horseman a second time. I was able to click the dirt pile, but nothing.

However, the event worked extremely well this time. The head worked correctly flying around, the animations were correct, the pumpkin mob assistants appeared, and I was able to kill the horseman  and gain the achievement. My apologies to Xinef, but I'm can't explain the issues that occurred before.

I'd say there are just a few small bugs here resetting the horseman after he wipes the party and to correct the ability to call him back again at a later time if you leave the instance after dying.
